pcsx2/ImGui/FullscreenUI.cpp

```diff
@@ -4297,25 +4325,123 @@ void FullscreenUI::DrawPatchesOrCheatsSettingsPage(bool cheats)
 		}
 	}
 
-	for (const Patch::PatchInfo& pi : patch_list)
+
+
+	struct PatchGroup
 	{
-		const auto enable_it = std::find(enable_list.begin(), enable_list.end(), pi.name);
+		bool bUncategorizedContents = false;
+		std::vector<std::string>		headers;			//	Total Headers
+		std::vector<Patch::PatchInfo>	pInfoList;			//	All Patches
+	};
+	PatchGroup s_patch_group;
 
-		bool state = (enable_it != enable_list.end());
-		if (ToggleButton(pi.name.c_str(), pi.description.c_str(), &state, master_enable))
+	//	Organize patch list content
+	for (const Patch::PatchInfo& pi : patch_list)
+	{
+		auto header = pi.group;
+		s_patch_group.pInfoList.push_back(pi);
+		if (header != "")
 		{
-			if (state)
+			auto result = find(s_patch_group.headers.begin(), s_patch_group.headers.end(), header);
+			if (result == s_patch_group.headers.end())
+				s_patch_group.headers.push_back(header);
+		}	
+		else
+			s_patch_group.bUncategorizedContents = true;
+
+		/*
```

Comment:
Unused code like this should be removed.

Reply:
> Unused code like this should be removed.


Result. As you can see the uncategorized patches are no longer populated

![image](https://github.com/PCSX2/pcsx2/assets/80198020/3ca26120-5d9d-41ec-9626-e204a83042f6)

Reply:
You missed my reply below - I was referring to the commented out block, not this line specifically. Either I tagged the wrong line, or GitHub decided to expand the view up a bit.